src/aui-form-builder/js/aui-form-builder-field-list.js

```diff
@@ -172,6 +172,16 @@ A.FormBuilderFieldList  = A.Base.create('form-builder-field-list', A.Base, [], {
             container.append(field.get('content'));
         });
 
+        setTimeout(function() {
```

Comment:
I'm not sure why you need the timeout here, is it waiting for some other code to run on other classes like the form builder? If it's waiting for something like that then you could listen to this event for when `fields` change on the form builder instead, since it has access to all field list instances on its layout, I think it's actually already a target for their events. Then it would be easier to control when this code needs to run.

If that's not the case let me know why you need this timeout for now then so we can figure out a better way to do this.